recipes/minikube/meta.yaml

```
@@ -0,0 +1,51 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "minikube" %}
+{% set version = "v1.3.1" %}
```

Remove `v` from the version number and add it to the URL:
```suggestion
{% set version = "1.3.1" %}
```

---

sounds good. thanks.